test/ex_machina/ecto_test.exs

```diff
@@ -100,9 +142,10 @@ defmodule ExMachina.EctoTest do
 
     account = ExMachina.Ecto.assoc(MyApp.EctoFactories, attrs, :account, factory: :user)
 
-    newly_created_account = %{id: 3, admin: false, name: "John Doe"}
+    vals = %{admin: false, name: "John Doe"}
+    newly_created_account = struct(MyApp.User, vals) |> Map.put(:id, 1)
```

Comment:
Actually I think it would make sense to just create a struct and remove the `vars` variable altogether on this test and the one above it

```
newly_created_user = %MyApp.User{id: 1, name: "John Doe", admin: false}
```

What do you think?